extension/chrome/settings/modules/compatibility.htm

```diff
@@ -16,6 +16,14 @@ <h1>OpenPGP key compatibility test</h1>
 
       <div class="line">Locally test any public or private key.</div>
 
+      <div class="line">
+        <label
+          >Search for public key on FlowCrypt Attester
+          <input type="text" class="input_email" placeholder="Email address" maxlength="256" />
```

Comment:
let's add `style="height: 37px !important"`, so it'll be the same height as button and passphrase input

Reply:
Thank you very much.

![image](https://github.com/FlowCrypt/flowcrypt-browser/assets/46025304/3b534fbe-4fa9-4d54-87ea-40ecb0f4c7c9)